core/client/verify.go

```diff
@@ -73,7 +70,7 @@ func New(vrf vrf.PublicKey,
 //  - Verify consistency proof from log.Root().
 //  - Verify inclusion proof.
 func (v *Verifier) VerifyGetEntryResponse(ctx context.Context, domainID, appID, userID string,
-	trusted *trillian.SignedLogRoot, in *pb.GetEntryResponse) error {
+	trusted trillian.SignedLogRoot, in *pb.GetEntryResponse) error {
```

Comment:
"trusted" sounds like a bool.

Reply:
Why are you taking a copy of trillian.SignedLogRoot?

Reply:
Because I don't want this function to be able to modify the argument.

Reply:
This function *could* modify parts of the argument even with this change, since `trusted.RootHash` and `trusted.Signature` are pointers.